projects/core/components/error/error.style.less

```diff
@@ -9,7 +9,6 @@
 
 .t-message-text {
     margin-top: 0.25rem;
-    white-space: pre-wrap;
```

Comment:
We need it so people can use `\n` inside string errors. I believe `white-space: pre-line;` was meant to be used.